### PR TITLE
Remove card-background-color from doc and story

### PR DESCRIPTION
**Pull request by fcollonval**
# Pull Request

## 📖 Description

In https://github.com/microsoft/fast/pull/4759 the `card-background-color` has been removed. This removes mention of it in the README and in the story.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components/src/card/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # fast-card
-The `fast-card` component is a visual container and design system provider. By default `fast-card` applies `neutralFillCard` to its background that is calculated from its parent design system provider. If a custom background color is desired the attribute `card-background-color` is available, this will reset the cards design system to that value. Cards are snapshots of content that are typically used in a group to present collections of related information.
+The `fast-card` component is a visual container and design system provider. By default `fast-card` applies `neutralFillCard` to its background that is calculated from its parent design system provider. Cards are snapshots of content that are typically used in a group to present collections of related information.
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/card/README.md:2]:
```suggestion
The `fast-card` component is a visual container. By default `fast-card` applies `neutralFillLayerRecipe` to its background that is calculated from its parent's fill color. Cards are snapshots of content that are typically used in a group to present collections of related information.
```

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]
I think there might be other details changed. I need @bheston to review please.

File: packages/web-components/fast-components/src/card/README.md (hunk 1 above)
Comment (bheston) [reply packages/web-components/fast-components/src/card/README.md:2]: Looks great.

**Review by bheston — APPROVED** [review]

File: packages/web-components/fast-components/src/card/README.md (hunk 1 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/card/README.md:2]: I'm going to commit this and rebuild so we can get this in.

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**